fineract-command/src/test/resources/org/apache/fineract/messages.properties

```diff
@@ -69,3 +69,7 @@ org.apache.fineract.common.validation.time.DurationMax.message=must be shorter t
 org.apache.fineract.common.validation.time.DurationMin.message=must be longer than${inclusive == true ? ' or equal to' : ''}${days == 0 ? '' : days == 1 ? ' 1 day' : ' ' += days += ' days'}${hours == 0 ? '' : hours == 1 ? ' 1 hour' : ' ' += hours += ' hours'}${minutes == 0 ? '' : minutes == 1 ? ' 1 minute' : ' ' += minutes += ' minutes'}${seconds == 0 ? '' : seconds == 1 ? ' 1 second' : ' ' += seconds += ' seconds'}${millis == 0 ? '' : millis == 1 ? ' 1 milli' : ' ' += millis += ' millis'}${nanos == 0 ? '' : nanos == 1 ? ' 1 nano' : ' ' += nanos += ' nanos'}
 # dummy
 org.apache.fineract.dummy.request.content.not-empty=Dummy request content must have a value
+
+# Interest Rate Chart validation messages
```

Comment:
Why these validation messages are duplicated? (fineract-command and fineract-provider module)